### DISCUSSION TITLE
Why is my timescale is weirdly formatted by default?

### DISCUSSION THREAD

**Discussion by sushanyadav** [tradingview/lightweight-charts#1207] [Q&A] [answered]
My chart data looks like this:

```
[
    {
        "time": 1669334400000,
        "value": 4.451876035174693
    },
    {
        "time": 1669420800000,
        "value": 4.4347502296225345
    },
    {
        "time": 1669436738000,
        "value": 4.760296943556378
    }
]
```

![CleanShot 2022-11-26 at 10 21 52](https://user-images.githubusercontent.com/53527664/204072451-baab1cce-0960-4608-814b-9c75cc11c76d.png)

--
Or help me format the tick in this format:
![CleanShot 2022-11-26 at 11 16 05](https://user-images.githubusercontent.com/53527664/204073812-ccd00649-c515-4f87-978b-2fb24aaee7a8.png)

**Comment by romfrancois** [comment, marked as answer]
Hi @sushanyadav !

Try formatting time in epoch timestamp vs simple timestamp (aka divide your time by 1000).

There's 3 ways time values can be used:

```
time: '2019-01-18'
```

or 

```
time: {year: 2018, month: 9, day: 23}
```

or 

```
time: 1642427876
```


If it helps have a look at some of our [examples](https://www.tradingview.com/lightweight-charts/) or [documentation](https://tradingview.github.io/lightweight-charts/tutorials).
